### ISSUE TITLE
Expected to find a 'className' configured for the exporter.

### ISSUE THREAD

**Issue by filipecampos** [camunda/camunda#23459]
In the sequence of [fix: Throw IllegalArgumentException when non-existing exporters are configured #23074](https://github.com/camunda/camunda/pull/23074), there are error reports from test clusters in SaaS that complain about a missing classname for the ES exporter:
[https://console.cloud.google.com/errors/detail/CIupyayGq7jQAQ;service=zeebe;time=P7D;locations=global?project=camunda-saas-int-chaos](https://console.cloud.google.com/errors/detail/CIupyayGq7jQAQ;service=zeebe;time=P7D;locations=global?project=camunda-saas-int-chaos)
[https://console.cloud.google.com/errors/detail/CIupyayGq7jQAQ;service=zeebe;time=P7D;locations=global?project=camunda-saas-int](https://console.cloud.google.com/errors/detail/CIupyayGq7jQAQ;service=zeebe;time=P7D;locations=global?project=camunda-saas-int)

It seems that the broker on initialization fails to pick the configuration for the exporter class name. As seen in the errors reported above, it only happens in some brokers.

Stack trace:

```
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'broker' defined in io.camunda.zeebe.broker.BrokerModuleConfiguration: Failed to instantiate [io.camunda.zeebe.broker.Broker]: Factory method 'broker' threw exception with message: Expected to find a 'className' configured for the exporter. Couldn't find a valid one for the following exporters [elasticsearch]
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:648) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:636) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1355) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1185) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:562) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:522) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:337) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:335) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:975) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:971) ~[spring-context-6.1.13.jar:6.1.13]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:625) ~[spring-context-6.1.13.jar:6.1.13]
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146) ~[spring-boot-3.3.4.jar:3.3.4]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754) [spring-boot-3.3.4.jar:3.3.4]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:456) [spring-boot-3.3.4.jar:3.3.4]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:335) [spring-boot-3.3.4.jar:3.3.4]
	at io.camunda.application.StandaloneBroker.main(StandaloneBroker.java:30) [camunda-zeebe-8.7.0-SNAPSHOT.jar:8.7.0-SNAPSHOT]
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [io.camunda.zeebe.broker.Broker]: Factory method 'broker' threw exception with message: Expected to find a 'className' configured for the exporter. Couldn't find a valid one for the following exporters [elasticsearch]
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:178) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:644) ~[spring-beans-6.1.13.jar:6.1.13]
	... 17 more
Caused by: java.lang.IllegalArgumentException: Expected to find a 'className' configured for the exporter. Couldn't find a valid one for the following exporters [elasticsearch]
	at io.camunda.zeebe.broker.system.SystemContext.validateExporters(SystemContext.java:164) ~[zeebe-broker-8.7.0-SNAPSHOT.jar:8.7.0-SNAPSHOT]
	at io.camunda.zeebe.broker.system.SystemContext.validateConfiguration(SystemContext.java:117) ~[zeebe-broker-8.7.0-SNAPSHOT.jar:8.7.0-SNAPSHOT]
	at io.camunda.zeebe.broker.system.SystemContext.initSystemContext(SystemContext.java:101) ~[zeebe-broker-8.7.0-SNAPSHOT.jar:8.7.0-SNAPSHOT]
	at io.camunda.zeebe.broker.system.SystemContext.<init>(SystemContext.java:81) ~[zeebe-broker-8.7.0-SNAPSHOT.jar:8.7.0-SNAPSHOT]
	at io.camunda.zeebe.broker.BrokerModuleConfiguration.broker(BrokerModuleConfiguration.java:94) ~[camunda-zeebe-8.7.0-SNAPSHOT.jar:8.7.0-SNAPSHOT]
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(Unknown Source) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:146) ~[spring-beans-6.1.13.jar:6.1.13]
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:644) ~[spring-beans-6.1.13.jar:6.1.13]
	... 17 more
```

**Comment by rodrigo-lourenco-lopes** [comment]
Important to point out that the configuration picked from the saas cluster was from the only zeebe pod that had initialized correctly. We could not pick the config from the other pods via the endpoint as these were crash looping. Therefore this  pod would not show the error anyway.

We tried to replicate the same issue by deploying the latest version from main in the benchmarks, and the variables were picked correctly. Additionally we created a cluster in saas using the same generation and same cluster type, and there the variables were picked correctly as well. So far it does not seem easy to replicate.

For now, scalling down and back again the stateful set for zeebe seems like a quick fix.

Generations that showed the issue: [temp_I3bWqdLFSF](https://accounts.cloud.ultrawombat.com/consoleadmin/generations/142d26f0-2dfc-4f07-8805-5264a0cea93d) and Generation: [Operate license tag test](https://accounts.cloud.ultrawombat.com/consoleadmin/generations/3ec948b5-66a5-46fb-ba56-0a94dde25ce7)
